IPython/core/magics/auto.py

```diff
@@ -16,7 +16,7 @@
 # Our own packages
 from IPython.core.magic import Bunch, Magics, magics_class, line_magic
 from IPython.testing.skipdoctest import skip_doctest
-from IPython.utils.warn import error
+from logging import error
```

Comment:
So this one will , on FLP (F@#$%g Legacy Python) try to import error from `./logging.py` in the same folder. Which fails.

It will likely need a `from __future__ import absolute_import` at the top of the file.